Arista/ninjaDump

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+import sys
+import shutil
+import os
+
+from os import listdir
+from os.path import isfile, join
+
+ninjaStatsDirectory=os.environ.get('NINJA_STATS_DIRECTORY')
+filesToIngest = os.environ.get('NINJA_STATS_MAX_FILES_TO_INGEST')
+
+if not ninjaStatsDirectory:
+   print 'NINJA_STATS_DIRECTORY not configured'
+   sys.exit(1)
+
+# A safety valve to stop stats collection
+if isfile(join(ninjaStatsDirectory, "DISABLED")):
+   print 'Ninja stats collection disabled'
+   sys.exit(0)
+
+if not filesToIngest:
+   filesToIngest = '10'
+filesToIngest = int( filesToIngest )
+
+files = [ f for f in listdir( ninjaStatsDirectory ) if isfile(
+         join( ninjaStatsDirectory, f ) ) ]
+
+files = files[:filesToIngest]
+
+for f in sorted( files ):
+   f = join( ninjaStatsDirectory, f )
+   with open( f, "r" ) as fd:
+      try:
+         shutil.copyfileobj( fd, sys.stdout )
```

Comment:
is the whole purpose of this plugin just to cat the dump files to stdout?
sounds shorter to do in bash:

find $dir -type f | sort | head -${max} | xargs cat

but upto you.